"GPIO4 was not declared in this scope"

Hey, thank you for the project! I'm having this error when trying to update my nspanel. Any clue?

```
In file included from src/esphome/components/nspanel_lovelace/nspanel_lovelace_upload_arduino.cpp:4:
src/esphome/components/nspanel_lovelace/nspanel_lovelace.h: In member function 'void esphome::nspanel_lovelace::NSPanelLovelace::soft_reset_display()':
src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:131:18: error: 'GPIO4' was not declared in this scope
     digitalWrite(GPIO4, 1);
                  ^~~~~
src/esphome/components/nspanel_lovelace/nspanel_lovelace.h:131:18: note: suggested alternative: 'GPIO'
     digitalWrite(GPIO4, 1);
                  ^~~~~
```

--- a/components/nspanel_lovelace/nspanel_lovelace.h
+++ b/components/nspanel_lovelace/nspanel_lovelace.h
@@ -128,9 +128,9 @@ public:
     vTaskDelay(pdMS_TO_TICKS(1000));
     gpio_set_level(GPIO_NUM_4, 0);
 #else
-    digitalWrite(GPIO4, 1);
+    digitalWrite(4, 1);
     delay(1000);
-    digitalWrite(GPIO4, 0);
+    digitalWrite(4, 0);
 #endif
   }
 

